official/resnet/resnet.py

```diff
@@ -192,9 +193,22 @@ def conv2d_fixed_padding(inputs, filters, kernel_size, strides, data_format):
       data_format=data_format)
 
 
+def group_block(f):
+  setattr(f, "block_counter", 0)
```

Comment:
add a docstring describing the function and why/how to use it. It would also be great if you can pass the block type as an argument ("building_v1" or "bottleneck_v2"), and add it to the variable scope name. The current screenshots look great!